straxen/plugins/peaks/peak_corrections.py

```diff
@@ -0,0 +1,125 @@
+from straxen.plugins.events.corrected_areas import CorrectedAreas
+import strax
+import numpy as np
+import straxen
+
+export, __all__ = strax.exporter()
+
+
+@export
+class PeakCorrectedAreas(CorrectedAreas):
+    __version__ = '0.0.1'
+
+    depends_on = ('peak_basics', 'peak_positions', 'peak_per_event')
+    data_kind = 'peaks'
+    provides = 'peak_corrections'
```

Comment:
Also, do you mind if I change this to `peak_corrections`? Usually, we need to use `peak` in plugins except `peaks`.